docs/Komenci ee781d07086748b0afa38e2cbb3d8fa3.md

```diff
@@ -0,0 +1,389 @@
+# Runbook
```

Comment:
Why such a weird file name `docs/Komenci ee781d07086748b0afa38e2cbb3d8fa3.md`? Is it intended?

Reply:
I did not notice, thanks for the catch! The name comes from the export from the internal version of the runbook.
Renamed to "runbook.md", which makes more sense.